### DISCUSSION TITLE
【求助】

### DISCUSSION THREAD

**Discussion by langxmo** [share121/inter-knot#673] [其他]
![58d7a510c4cdcd190ae3e03eadf074c2](https://github.com/user-attachments/assets/deebdef2-b40d-4926-a655-2aaf291007bc)
有没有绳匠懂得这是什么情况，一直没办法正常加载图片，哪位大佬指点一下，谢谢

**Comment by mhpmc** [comment]
是裸连吗？

**Reply by langxmo** [reply]
不是，裸连也差不多

**Comment by TrafficLight6** [comment]
我这也是这样，[看这](https://github.com/share121/inter-knot/discussions/691#new_comment_form)

**Comment by AusertDream** [comment]
部分图片可以加载出来，应该是加载速度慢的原因，是裸连的吗，是的话，试试看挂梯子吧

**Reply by langxmo** [reply]
我用的watt连的，连和不连几乎没什么区别

**Comment by share121** [comment]
修好了

**Comment by user1-cloud** [comment]
**_喜报_**！修好了
